contrib/podmanimage/machine/Containerfile

```diff
@@ -0,0 +1,15 @@
+FROM quay.io/centos-bootc/fedora-bootc:eln
+
+RUN dnf -y update; \
+rpm --setcaps shadow-utils 2>/dev/null; \
+dnf -y install podman subscription-manager crun crun-wasm crun-krun chrony \
```

Comment:
Why do we need skopeo and buildah?

Reply:
We don't but I think it is useful for debugging and difficult to install if they don't exist.

Reply:
They add 60M to the disk / container so I feel rather hesitant.

Reply:
Ok I can remove.

Reply:
Personally I've quite often wanted the ability to inspect a remote image without fetching it...and that's just in skopeo right?  (It would make sense to me to have something like this in podman too...`podman image remote-inspect`?  Dunno)

As far as buildah, I think it's basically because it's been promoted as a peer to `podman build`, and so it goes where podman goes.

Reply:
The funning thing, my story of skopeo being created is exactly this. we wanted docker inspect --remote.  So maybe it is time to add `podman image inspect --remote` One issue there is inspecting images versus manifest.

--raw versus default.  Also --remote might be misinterpreted by CLI.

Reply:
I this should be a separate issue.  Note that skopeo runs on Win and Mac as well.  So I don't think this must be part of the `podman machine` where IMO the focus should be on what Podman _needs_ rather than what some developers may find useful.  Ideally, users shouldn't need to ssh into the machine.